### DISCUSSION TITLE
binascii.Error: Non-hexadecimal digit found

### DISCUSSION THREAD

**Discussion by isidororeyes** [coccoinomane/crabada.py#95] [Q&A] [answered]
Hello Team,

I'm having issues when sending a team to mine. In the env file I've set the USER_1_ADDRESS = 0x[......]
and also from command line python -m bin.mining.sendTeamsMining 0x[....]

and I keep getting the binascii.Error: Non-hexadecimal digit foun

I'm sure there are no extra chars from command line nor in the file. I'm positive the address is correct.
I've tried also all caps for the address and same result.

If the env file is upper case and command line call is lower case I get this error
src.common.exceptions.UserException: User address not registered: 0xdc886f4

Any idea of what should be the casing and where?
Thanks!

EDIT by @coccoinomane: wallet reference.

**Comment by coccoinomane** [comment, marked as answer]
Hi @isidororeyes !

This error might happen if you use your mnemonic seed key instead of your private key.

For more details, please see https://github.com/coccoinomane/crabada.py/issues/88.

Cheers,
Cocco
